code/core/src/manager/components/sidebar/SidebarBottom.tsx

```diff
@@ -180,16 +180,18 @@ export const SidebarBottomBase = ({ api, notifications = [], status = {} }: Side
   );
 };
 
+const TESTING_MODULE_ID = 'storybook-testing-module';
+
 export const SidebarBottom = () => {
   const api = useStorybookApi();
   const { notifications, status } = useStorybookState();
 
   useEffect(() => {
     api.getChannel()?.on(TESTING_MODULE_RUN_PROGRESS_RESPONSE, (data) => {
       if ('payload' in data) {
-        // console.log('progress', data);
+        console.log('progress', data);
```

Comment:
Console log still necessary?

Reply:
no! it's always useful though, so we keep uncommenting and commenting. I will comment it back